htlcswitch/link.go

```diff
@@ -760,6 +818,26 @@ func (l *channelLink) syncChanStates() error {
 			}
 		}
 
+		// If we have been signaled to retransmit Shutdown, we'll do so
+		// here.
+		if l.cfg.RetransmitShutdown {
+			// Recreate the Shutdown message.
+			shutdownMsg := lnwire.NewShutdown(
+				l.ChanID(),
+				l.channel.LocalUpfrontShutdownScript(),
+			)
+
+			// Flip the shutdownInit and shutdownSent bools.
```

Comment:
are we allowed to init and send the msg multiple times?

Reply:
it's undefined in the spec meaning anything can happen (peer disconnects or something else) but the code here shouldn't be sending `shutdown` multiple times